Files/Dialogs/AddItemDialog.xaml

```diff
@@ -13,6 +13,7 @@
     BorderThickness="0"
     CornerRadius="4"
     PrimaryButtonText="Cancel"
+    PrimaryButtonStyle="{StaticResource DefaultButtonStyle}"
```

Comment:
```suggestion
    PrimaryButtonStyle="{StaticResource AccentButtonStyle}"
```

Reply:
The button in AddItemDialog didn't have primary color previously, so I didn't change it.

Reply:
That must have been an oversight.